mlflow/server/js/src/experiment-tracking/components/ArtifactView.tsx

```diff
@@ -375,6 +375,14 @@ export class ArtifactViewImpl extends Component<ArtifactViewImplProps, ArtifactV
     return this.props.artifactRootUri;
   }
 
+  getActiveNodeFilePath() {
+    if (this.state.activeNodeId) {
+      const node = ArtifactUtils.findChild(this.props.artifactNode, this.state.activeNodeId);
+      return node.fileInfo.path;
```

Comment:
I wonder if we should wrap it with try/catch since `ArtifactUtils.findChild` can throw an error

Reply:
I copied this kind of logic from existing functions like `getActiveNodeRealPath`, `getActiveNodeSize` , `activeNodeIsDirectory`. They all check for `this.state.activeNodeId` before calling `ArtifactUtils.findChild` - perhaps this prevents from encountering the error case? I'm not familiar with this code.

Note I'm only calling this new `getActiveNodeFilePath` function alongside the existing call to `activeNodeIsDirectory` (inside `componentDidUpdate`), so I'm not adding a new kind of error path (if the error would be thrown it would already be thrown while calling `activeNodeIsDirectory`).